semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -51,12 +51,13 @@ def exclude_file_list(folder: str, pattern: str) -> List[str]:
     return file_names
 
 
-def filter_semconv(semconv, type_filter):
-    if type_filter:
+def filter_semconv(semconv, types_filter):
+    if types_filter:
+        types = types_filter.replace(" ", "").split(",")
```

Comment:
It could also make sense to validate that all the group type names are valid (known). Otherwise, typos in group type names here will lead to the group being skipped

Reply:
sure, done